polyfills/common/webapi_poly_common.js

```diff
@@ -265,7 +273,11 @@
           data: extraData,
           processAnswer: answer => {
             if (answer.event) {
-              listenerCb(answer.event, _listeners[extraData.type][reqId]);
+              if (typeof listenerCb === 'function') {
+                listenerCb(answer.event, _listeners[extraData.type][reqId]);
+              } else {
+                _listeners[extraData.type][reqId](answer.event);
```

Comment:
Don't we need to check if _listeners[extraData.type] exists? (and same for [reqId])